src/App.test.jsx

```diff
@@ -65,14 +64,8 @@ describe('App should render a searchable list of pokemon', () => {
     expect(searchBar.value).toBe('b');
     // const images2 = await screen.findAllByRole('img', {timeout: 5000});
     // expect(images2.length).toEqual(1);
-    await waitForElementToBeRemoved(await screen.findByText(/venusaur/i), { timeout: 5000 });
-    // waitFor(() => {
-      
-    //   screen.debug();
-
-    // }, {timeout: 5000})
-    // const results = await screen.findAllByRole('img', {timeout: 5000});
-    // expect(results.length).toEqual(1);
+    await waitForElementToBeRemoved(await screen.findByText(/venusaur/i), { timeout: 2000 });
+    
```

Comment:
We are super close here. We will also want to test for the element on the page to be 'butterfree' after searching. Otherwise, fantastic job!